RapidReact/src/main/java/frc/robot/subsystem/ClimberSubsystem.java

```diff
@@ -329,6 +330,34 @@ public void periodic() {
     climberRightPosition.log(LogLevel.GENERAL, climberRight.getSelectedSensorPosition());
     climberRightError.log(LogLevel.GENERAL, climberRight.getClosedLoopError());
 
+    boolean climbLeftRevLimitSwitchClosed = climberLeft.getSensorCollection().isRevLimitSwitchClosed();
+    boolean climbRightRevLimitSwitchClosed = climberRight.getSensorCollection().isRevLimitSwitchClosed();
+
+    if (!climbLeftRevLimitSwitchClosed)
+    {
+      if (!climbLeftEncoderZeroed)
+      {
+        climberLeft.set(ControlMode.PercentOutput, -0.1);
```

Comment:
Don't hard code this motor value. Call it a constant like "CLIMBER_RETRACT_SLOW * CLIMBER_LEFT_MOTOR_INVERTED".

Reply:
what's "CLIMBER_LEFT_MOTOR_INVERTED" mean in this example?

Reply:
Whether a positive voltage extends or retracts the climber (so either -1 or 1)